### ISSUE TITLE
Accessibility: aria-label attributes for buttons

### ISSUE THREAD

**Issue by seirerman** [fancyapps/fancybox#2317]
The buttons have title attributes which are ignored by most screen readers. The content of the buttons (SVGs) are also not readable.

Using the title attribute for users without disabilities _and_ aria-label attribute for those with disabilities would solve this problem:

`zoom: '<button data-fancybox-zoom class="..." title="{{ZOOM}}" aria-label="{{ZOOM}}">' +`

https://developers.google.com/web/tools/lighthouse/audits/button-name
